.travis.yml

```diff
@@ -52,7 +53,10 @@ script:
   - set -e # make this script fail as soon as any individual command fail
   - . ./scripts/docker/docker_rc.sh
   - dr make confload-"$TEMPLATE"
-  - run dr make
+  - if [ "$TEMPLATE" == "x86/test/qt-vnc" ] || [ "$TEMPLATE" == "x86/test/fs" ];
+    then run dr 'make > /dev/null';
```

Comment:
Are you sure we want drop build logs? I suspect build error will not be uncommon, changes to break these templates are even higher than others, considering number of modules included

Reply:
It's due to Travis log size limitations. If you just do `make`, then Travis failed because of log size exceeded.

Reply:
The log size exceded initially for `qt-vnc` only. But after updating 14.04 up to 16.04 test/fs log becames larger too.

Reply:
(posting to correct thread)
How about to store whole log and `tail` after make finished? (successfully or not). We can output significant number of lines without bothering travlis log limit then.

Reply:
Or we can echoing something like "Building..." every minute or smth. for example during the build (for Travis only).

Reply:
Seems like we assume something is regularly printed to stderr?
The suggestion may work. It like it way more than dropping the output. I suppose, can be implemented for all templates? There is a quirk there, it would hide any parallel buld issues that may be resolved by second make. Then we'll have to make sure exit code of whole expression is the one from first make

Reply:
I found [here](https://docs.travis-ci.com/user/common-build-problems/#build-times-out-because-no-output-was-received) "If you have a command that doesn’t produce output for more than 10 minutes, you can prefix it with travis_wait, a function that’s exported by our build environment".
So probably we could use this, and the output last 3Mb bytes. I will check this a bit later.

Reply:
I tried this in https://github.com/embox/embox/pull/1277/commits/d25066a155208e17b17d1be0251a2f3ef21b4b70. Seems to be working.